.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
+      - name: Docker login
```

Comment:
@rogh

Why do we need to do Docker login?

Reply:
oh woops, that was from another project. You don't. It's only if you want to create container images and push them to a registry. I'll remove.

Reply:
Thank you!